python/ray/tune/analysis/experiment_analysis.py

```diff
@@ -235,14 +253,22 @@ class ExperimentAnalysis(Analysis):
             Experiment.local_dir/Experiment.name/experiment_state.json
         trials (list|None): List of trials that can be accessed via
             `analysis.trials`.
+        default_metric (str): Default metric for `get_best_trial()` and its
+            derivatives.
+        default_mode (str): Default mode for `get_best_trial()` and its
+            derivatives.
```

Comment:
hmm I don't know what 'derivatives' mean - maybe something like "Default optimization metric used in the Analysis object for comparing results. This can be overriden ..."